### ISSUE TITLE
repeatSet and shuffle button not work in Chinese environment

### ISSUE THREAD

**Issue by Kleistr** [PsykoSoldi3r/vscode-itunes#20]
**Description**

I find that repeat & shuffle button doesn't work in Chinese environment.
I think these scripts may not work in other language environment.

**macOS version**

v10.14.6 (18G87)

**Fix**

**These changes only work in Chinese environment:**

repeatSetAll:
``` applescript
tell application "System Events" to tell process "iTunes"'s menu bar 1's menu bar item "控制"'s menu 1's menu item "重复"'s menu 1
	perform action "AXPress" of menu item "全部"
end tell
```
repeatSetOff:
``` applescript
tell application "System Events" to tell process "iTunes"'s menu bar 1's menu bar item "控制"'s menu 1's menu item "重复"'s menu 1
	perform action "AXPress" of menu item "关"
end tell
```
repeatSetOne:
``` applescript
tell application "System Events" to tell process "iTunes"'s menu bar 1's menu bar item "控制"'s menu 1's menu item "重复"'s menu 1
	perform action "AXPress" of menu item "单曲"
end tell
```

shuffleOff:
``` applescript
tell application "System Events" to tell process "iTunes"'s menu bar 1's menu bar item "控制"'s menu 1's menu item "随机播放"'s menu 1
	perform action "AXPress" of menu item "关闭"
end tell
```

shuffleOn:
``` applescript
tell application "System Events" to tell process "iTunes"'s menu bar 1's menu bar item "控制"'s menu 1's menu item "随机播放"'s menu 1
	perform action "AXPress" of menu item "开"
end tell
```

**Comment by joeykamsteeg** [comment]
Does you or someone know how to handle with unicode characters in Typescript? I run into a unicode problem, that Visual Studio Code doesn't recognise the correct scrips.

I have added multi language support for Dutch as well and published to Marketplace as version 0.13.0

**Comment by Kleistr** [comment]
Sorry, I have no idea about unicode. 

I update to the newest version, the Chinese characters changed (e.g. "控制" became "??") . All the Chinese characters became "?", and that caused these scripts doesn't work.

<img  width="812" alt="截屏2019-10-23上午1 55 54" src="https://user-images.githubusercontent.com/13588942/67315002-453b5200-f538-11e9-81d5-d51b248559a3.png">

**Comment by joeykamsteeg** [comment]
Fixed.

Problem seems to be by copy and pasting it into Visual Studio Code and saving the file. Have now edited the scripts with TextEdit and everything works fine.

Released in version 0.14.1
